src/main/resources/static/signin.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
Rename branch and change pull request description as this file is from another task.